pkgs/applications/misc/gnome-multi-writer/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv
+, fetchurl
+, appstream-glib
+, desktop-file-utils
+, gettext
+, glib
+, gnome3
+, gtk3
+, gusb
+, libcanberra-gtk3
+, libgudev
+, meson
+, ninja
+, pkgconfig
+, wrapGAppsHook
+, polkit
+, udisks
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-multi-writer";
+  version = "3.32.1";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1apdd8yi12zagf82k376a9wmdm27wzwdxpm2wf2pnwkaf786rmdw";
+  };
+
+  nativeBuildInputs = [
```

Comment:
we need to add `hicolor-icon-theme` for the setup hook
https://gitlab.gnome.org/GNOME/gnome-multi-writer/blob/master/meson_post_install.sh#L11

Reply:
also believe if you add `docbook2x` we can build the man pages 
* https://gitlab.gnome.org/GNOME/gnome-multi-writer/blob/master/man/meson.build#L2

Reply:
IIRC, docbook2x provides a different docbook2man than Debian does, resulting in an error. I will try to add the setup hook later.

Reply:
Hmm, I wondered why do we have this in `hicolor-icon-theme` when it is ussed by GTK only; and actually, Qt [implemented](https://woboq.com/blog/qicon-reads-gtk-icon-cache-in-qt57.html) support for these files. But still, since `gtk-update-icon-cache` is part of GTK, should not the clean up script be there as well?

Reply:
> But still, since gtk-update-icon-cache is part of GTK, should not the clean up script be there as well

I'd agree
> Do you think it would make sense to actually move the setup hook to gtk?

You need the tool from gtk to generate it even for qt so it's probably best for it to be there.

Reply:
Merging as this will be solved by https://github.com/NixOS/nixpkgs/pull/68552